src/Notification/NotificationServiceProvider.php

```diff
@@ -30,34 +30,43 @@ public function boot()
      */
     public function registerNotificationTypes()
     {
-        $blueprints = [
-            DiscussionRenamedBlueprint::class => ['alert']
-        ];
+        $this->app->singleton('flarum.notification.blueprints', function () {
```

Comment:
As a side note, this setup is great, because it will allow us to edit the enabled-by-default settings of some types.